packs/data/equipment.db/ghast-stiletto.json

```diff
@@ -0,0 +1,120 @@
+{
+    "_id": "VSxLv1JRSo10waL0",
+    "img": "systems/pf2e/icons/equipment/weapons/dagger.webp",
+    "name": "Ghast Stiletto",
+    "system": {
+        "MAP": {
+            "value": ""
+        },
+        "baseItem": "dagger",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "category": "simple",
+        "containerId": null,
+        "damage": {
+            "damageType": "piercing",
+            "dice": 1,
+            "die": "d4"
+        },
+        "description": {
+            "value": "<p>A <em>ghast stiletto</em> is a <em>+1 dagger</em> wrapped in the still-undead skin of a ghast. While you wield the dagger, you gain a +1 status bonus to all saves against paralysis.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> command, Interact (divine, necromancy, olfactory)</p>\n<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p><strong>Effect</strong> The ghast stiletto afflicts the next living target struck by the dagger with a terrible wound that emits the foul stench of the grave. For the next minute, the target creature, as well as any creatures within a @Template[type:emanation|distance:10], must succeed at a @Check[type:fortitude|dc:16|traits:divine,necromancy,olfactory] save or become @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 1} (plus @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 1} as long as it's sickened on a critical failure). While within the emanation, the creature takes a –2 circumstance penalty to saves against disease and to recover from the sickened condition. A creature that succeeds at its save is temporarily immune for 1 minute.</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include skin from a ghast.</p>"
```

Comment:
The -2 penalty sounds like the Ghast Stench effect. Maybe we can add an aura effect to give to the target?

Reply:
We don't have any automation for retching as is.